x-pack/plugins/asset_manager/server/lib/accessors/hosts/index.ts

```diff
@@ -5,15 +5,15 @@
  * 2.0.
  */
 
-import type { AssetClientDependencies } from '../../../types';
-import type { GetHostsOptionsPublic } from '../../../../common/types_client';
-import type { OptionsWithInjectedValues } from '..';
+import { validateESDate } from '../../validators/validate_es_date';
+import type { GetHostsOptions, GetHostsOptionsInjected } from './shared_types';
+export { getHosts } from './get_hosts';
 
-export type GetHostsOptions = GetHostsOptionsPublic & AssetClientDependencies;
-export type GetHostsOptionsInjected = OptionsWithInjectedValues<GetHostsOptions>;
+export type { GetHostsOptions, GetHostsOptionsInjected };
 
-export interface HostIdentifier {
-  'asset.ean': string;
-  'asset.id': string;
-  'asset.name'?: string;
+export function validateGetHostsOptions(options: GetHostsOptions) {
```

Comment:
function can also verify that `from` is before `to`

Reply:
Good idea

Reply:
I simplified things a bit since there is only one version of each function now, which also simplified validation. https://github.com/elastic/kibana/pull/167692/commits/34c6d1af13a5cac6cc3932c9d185b8c380db0c8e